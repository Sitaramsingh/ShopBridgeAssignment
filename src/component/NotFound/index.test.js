import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './index';


describe('Component should be render ', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<NotFound/>);
    expect(component).toMatchSnapshot();
  });
});