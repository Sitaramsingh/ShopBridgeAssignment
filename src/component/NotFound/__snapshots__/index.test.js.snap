// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be render  should render correctly in "debug" mode 1`] = `
<div
  className="not-found"
>
  <h1
    className="display-1"
  >
    Page Not Found
  </h1>
</div>
`;
